qualtran/_infra/controlled.py

```diff
@@ -171,6 +185,51 @@ def __eq__(self, other: Any) -> bool:
     def __hash__(self):
         return hash((self.qdtypes, self.shapes, self._cvs_tuple))
 
+    def to_cirq_cv(self) -> cirq.SumOfProducts:
+        """Convert CtrlSpec to cirq.SumOfProducts representation of control values."""
+        cirq_cv = []
+        for qdtype, cv in zip(self.qdtypes, self.cvs):
+            for idx in Register('', qdtype, cv.shape).all_idxs():
+                cirq_cv += [*qdtype.to_bits(cv[idx])]
+        return cirq.SumOfProducts([tuple(cirq_cv)])
+
+    @classmethod
+    def from_cirq_cv(
+        cls,
+        cirq_cv: cirq.ops.AbstractControlValues,
```

Comment:
this isn't in the top-level cirq namespace?

Reply:
nope